topic: "/manual_platform_control/joy"
base:
  linear_x: {axis: 0, scale: 0.2}
  linear_y: {axis: 1, scale: 0.1}
  angular_z: {axis: 2, scale: -0.4}
deadzone: 0.0 #axis values smaller than this value will be set to 0.0. For a spacenav, its hard to control only 1 axis